tests/raw_query/models.py

```diff
@@ -35,5 +35,9 @@ class Reviewer(models.Model):
     reviewed = models.ManyToManyField(Book)
 
 
+class Product(models.Model):
+    price = models.DecimalField(max_digits=10, decimal_places=2)
+
```

Comment:
Should I give it `default=0, blank=True` to avoid making changes in places where `Coffee` is already being used ?